tools/chplcheck/src/rules.py

```diff
@@ -36,6 +36,32 @@ def variables(node: AstNode):
             yield from variables(child)
 
 
+def might_incorrectly_report_location(node: AstNode) -> bool:
+    """
+    Some Dyno AST nodes do not return locations in the way that we expect.
+    For instance, some NamedDecl nodes currently use the name as the location,
+    which does not indicate their actual indentation. Rules that depend on
+    indentation should leave these variables alone.
+    """
+
+
+    # some NamedDecl nodes currently use the name as the location, which
+    # does not indicate their actual indentation.
```

Comment:
There should be an issue for this as well, can you add the link here like 'private'?

If there is no issue, please open one 😄